zip-0317.rst

```diff
@@ -55,7 +55,7 @@ usage of the chain has been characterized by high-output transactions with
 1,100 outputs, paying the same conventional fee as a transaction with 2 outputs.
 
 The objective of the new fee policy, once it is enforced, is for fees paid by
-transactions to fairly reflect the processing costs that their inputs and outputs
+transactions to be proportional to the processing costs that their inputs and outputs
```

Comment:
The objective was and is for fees paid by transactions to "fairly reflect" their costs. Setting them to "be proportional to" an estimate of the costs is *how* the policy does that. This is a Motivation section, so the "why" is more important than the "how" or "what" (which is already in the title of the ZIP anyway).

It is fair to say that whether a fee policy is "fair" is quite subjective. Still, we succeeded: the sandblasting stopped and no legitimate user has to pay excessive fees; and that was not by coincidence but by careful design.